pkg/binding/annotationmapper.go

```diff
@@ -45,26 +47,43 @@ func NewDefinitionBuilder(annotationName string, annotationValue string, configM
 	}
 }
 
-func (m *annotationBackedDefinitionBuilder) outputName() (string, error) {
-	// bail out in the case the annotation name doesn't start with "service.binding"
-	if m.name != AnnotationPrefix && !strings.HasPrefix(m.name, AnnotationPrefix+"/") {
-		return "", fmt.Errorf("can't process annotation with name %q", m.name)
+func (m *annotationBackedDefinitionBuilder) isServiceBindingAnnotation() (bool, error) {
+	if m.name == ProvisionedServiceAnnotationKey {
+		return false, nil
+	}
+
+	if m.name == AnnotationPrefix {
+		return true, nil
+	} else if strings.HasPrefix(m.name, AnnotationPrefix) {
+		if strings.HasPrefix(m.name, AnnotationPrefix+"/") {
+			return true, nil
+		}
+
+		// it starts with service.binding but doesn't have a trailing /, so it's not valid
+		return false, fmt.Errorf("can't process annotation with name %q", m.name)
```

Comment:
this was valid annotation until now, see https://github.com/redhat-developer/service-binding-operator/blob/3017eb443427396252959298b1b8e1ed923f29bd/test/acceptance/features/bindAppToService.feature#L31

Reply:
It's just the comment that's wrong, the behavior of the code is correct.  I'll update the comment.